packages/styles/.editorconfig

```diff
@@ -1,15 +1,4 @@
 # http://editorconfig.org
-root = true
-
-[*]
-charset = utf-8
-indent_style = space
-indent_size = 2
-insert_final_newline = true
-trim_trailing_whitespace = true
-end_of_line = lf
-
 
 [*.md]
 max_line_length = 0
```

Comment:
Do we even need this? If not, the file can be deleted entirely.

Reply:
I did not check if the properties are really needed, but only compared the editorconfig files.
Actually this property doesn't seem to do anything in our code.
But to not break anything, I moved the property to the main editorconfig and set the value to "off" as described in the documentation:
https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#max_line_length